#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_i386_optimize.dpatch by Andres Mejia <mcitadel@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to ensure i386 optimatizations only when building i386 Debian packages.
## This patch will work provided that the source tree was checked out to a directory named alienarena2007.

@DPATCH@
diff -urNad alienarena2007~/Makefile alienarena2007/Makefile
+++ alienarena2007/Makefile	2007-05-11 12:48:43.000000000 -0400
@@ -64,13 +64,13 @@
 	RELEASE_CFLAGS += -march=k8
 else
 ifeq ($(ARCH),i686)
-	RELEASE_CFLAGS += -march=i686
+	RELEASE_CFLAGS += -march=i386
 else
 ifeq ($(ARCH),i586)
-	RELEASE_CFLAGS += -march=i586
+	RELEASE_CFLAGS += -march=i386
 else
 ifeq ($(ARCH),i486)
-	RELEASE_CFLAGS += -march=i486
+	RELEASE_CFLAGS += -march=i386
 else
 ifeq ($(ARCH),i386)
 	RELEASE_CFLAGS += -march=i386
